internal/utility/jwt/jwt.go

```diff
@@ -0,0 +1,68 @@
+package jwt
+
+import (
+	"errors"
+	"fmt"
+	"math"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/spf13/viper"
+)
+
+type Claims struct {
+	jwt.StandardClaims
+	CountryCode string
+	CityID      string
+	IsGuest     bool
+}
+
+func CreateToken(id string, countryCode string, cityId string, isGuest bool) (string, error) {
+	// Create token claims
+	claims := Claims{
+		StandardClaims: jwt.StandardClaims{
+			ExpiresAt: math.MaxInt64,
```

Comment:
never expiring token. Is this right from security pov? 
or if it is just a dev thing, we can store it in config. For prod, we'll set it for shorter durations

Reply:
so reg. this, this is related to the functionality of the app, essentially we're not going to ask user for any username or password, once they launch the application they should be directly login (if any token found on their device) otherwise the application will create a new account, store the token locally on the device and login the user (by login, i mean they'll be able to jump directly to their profile homepage)

why not use refresh tokens?
if both refresh token and access token expires, then we'll need some sort of username/password mechanism to retrieve new Refresh token for user, since we are avoiding use of any username/password at all. hence used never-expiring tokens.

accounts in this application are going to be ephemeral

Reply:
makes sense